docs/rst/user_guide/data/srf.rst

```diff
@@ -0,0 +1,77 @@
+.. _sec-user_guide-data-srf:
+
+Spectral response function data sets
+====================================
+
+A spectral response function data set provide the spectral response of a
+given instrument on a specific platform and in a specific spectral band.
+
+Data sets download
+------------------
+
+All data sets are hosted on
+`https://eradiate.eu/data <https://eradiate.eu/data>`_.
+Find the data set that you need there and download it, or click the links below:
+
+* Sentinel-2
+
+  * `Sentinel-2A-MSI <https://eradiate.eu/data/sentinel_2a-msi.zip>`_
+    :cite:`ESA2017Sentinel2SpectralResponse`
+  * `Sentinel-2B-MSI <https://eradiate.eu/data/sentinel_2b-msi.zip>`_
+    :cite:`ESA2017Sentinel2SpectralResponse`
+
+* Sentinel-3
+
+  * `Sentinel-3A-SLSTR <https://eradiate.eu/data/sentinel_3a-slstr.zip>`_
+    :cite:`RAL2015Sentinel3ASLSTR`
+  * `Sentinel-3A-OLCI <https://eradiate.eu/data/sentinel_3a-olci.zip>`_
+    :cite:`Sentinel3CalValTeam2016OLCIASpectralResponse`
+  * `Sentinel-3B-SLSTR <https://eradiate.eu/data/sentinel_3b-slstr.zip>`_
+    :cite:`RAL2017Sentinel3BSLSTR`
+  * `Sentinel-3B-OLCI <https://eradiate.eu/data/sentinel_3b-olci.zip>`_
+    :cite:`Sentinel3CalValTeam2016OLCIBSpectralResponse`
+
+* Terra
+
+  * `Terra-MODIS <https://eradiate.eu/data/terra-modis.zip>`_
+    :cite:`MODIS2004`
```

Comment:
Any particular reason to ship those separately? Maintaining many links means more maintenance and higher chances for the data and docs to go out of sync, so unless you're sure it's a good idea, I'd advise to ship all SRFs as a  single bundle.

Reply:
I did not think about that. I don't see any cons to ship the data sets as a single archive file. The only pro that I can see to ship the data set separately is that you can choose to download only what you need. I don't think that's makes a great difference, given the size of the data sets. Let me re-upload the data sets in a single archive file and rebase and we can pull theses changes 😃

Reply:
Actually, I'll remove the links altogether because the SRF data sets are included in the eradiate-data submodule.